client/post-editor/editor-featured-image/style.scss

```diff
@@ -1,5 +1,20 @@
-.editor-featured-image__preview {
+.editor-featured-image {
+	display: block;
+	width: 100%;
+	padding: 0;
+}
+
+.editor-featured-image__current-image {
 	position: relative;
+	display: inline-block;
+	vertical-align: top;
+
+	.button.is-borderless & .gridicon {
```

Comment:
We should be able to avoid the nesting here if we add another className to the gridicon.